source/kraken/realtime.cpp

```diff
@@ -289,6 +335,8 @@ create_disruption(const std::string& id,
     if (trip_update.trip().HasExtension(kirin::contributor)) {
         disruption.contributor = trip_update.trip().GetExtension(kirin::contributor);
     }
+    bool has_effect = trip_update.HasExtension(kirin::effect);
```

Comment:
minor: It would be nice to have this line where it's being used (eg. 150 lines below :arrow_down: )